packages/messaging/src/controllers/window-controller.test.ts

```diff
@@ -36,12 +33,19 @@ import { WindowController } from './window-controller';
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
```

Comment:
License should be at the top of the file.

Reply:
hmm. I think I have a buggy format extension that moved it down. nice catch